vnext/Desktop.IntegrationTests/WebSocketModuleIntegrationTest.cpp

```diff
@@ -14,8 +13,8 @@ using std::make_unique;
 using std::unique_ptr;
 using std::vector;
 
-TEST_CLASS (WebSocketModuleIntegrationTest)
-{
+TEST_CLASS (WebSocketModuleIntegrationTest) {
+  // clang-format off
   TEST_METHOD(WebSocketModule_Ping)
```

Comment:
Not blocking, but did adding TEST_METHOD to ForEachMacros in `.clang-format` not work?

Reply:
Haven't tried it yet.
I'll push one more iteration, including the double null-check recommendation.